samples/README_Windows.md

```diff
@@ -0,0 +1,98 @@
+
+![Sample components diagram](sampledirstructure.png)
+
+All the samples that come with the Open Enclave SDK installation are all structured into two subdirectories (one for enclave and one for host) accordingly.
+
+| Files/dir        |  contents                                   |
+|:-----------------|---------------------------------------------|
+| enclave        | Files needed for building the sample enclave|
+| host           | Files needed for building the host          |
+
+### Prepare samples
+
+Building samples involves writing files into the working directory. You can do this in the directory in which you have installed your samples.
+You can also copy the samples from the location they were installed to another directory.
+
+```cmd
+xcopy  C:\openenclave\share\openenclave\samples C:\mysample
+```
+
+### How to build and run samples
+
+1. [x64 Native Tools Command Prompt for VS(2017 or 2019)](
+https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
+Normally this is accessible under the `Visual Studio 2019 or 2017` folder in the Start Menu.
+
+2. Set OpenEnclave_DIR to the cmake directory in the Open Enclave SDK installation.
+
+As an example, if the Open Enclave SDK is installed to `C:\openenclave`, then you would set OpenEnclaveDIR as shown below
+
+```cmd
+set OpenEnclaveDIR=C:\openenclave\lib\openenclave\cmake
+```
+
+3. To build a sample using CMake, change directory to your target sample directory and execute the following commands:
+
+```cmd
+mkdir build
+cd build
+cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\your\path\to\intel_and_dcap_nuget_packages
+ninja
+```
+
+4. To run the sample, use below:
+
+```cmd
+ninja run
+```
+
+## Samples
+
+The following samples demonstrate how to develop enclave applications using OE APIs. It's recommended to go through the following samples in the order listed.
+
+#### [HelloWorld](helloworld/README.md)
+
+- Minimum code needed for an OE app
+- Help understand the basic components an OE application
+- Demonstrate how to build, sign, and run an OE image
+
+#### [File-Encryptor](file-encryptor/README.md)
+
+- Shows how to encrypt and decrypt data inside an enclave
+- Uses AES mbedTLS API to perform encryption and decryption
+
+#### [Data-Sealing](data-sealing/README.md)
+
+- Introduce OE sealing and unsealing features 
+- Demonstrate how to use OE sealing APIs
+- Explore two supported seal polices
+  - OE_SEAL_POLICY_UNIQUE
+  - OE_SEAL_POLICY_PRODUCT
+
+#### [Remote Attestation](remote_attestation/README.md)
+
+- Explain how OE attestation works
+- Demonstrate an implementation of remote attestation between two enclaves running on different machines
+
+#### [Local Attestation](local_attestation/README.md)
+
+- Explain the concept of OE local attestation
+- Demonstrate an implementation of local attestation between two enclaves on the same VM
+
+#### [Switchless Calls](switchless/README.md)
```

Comment:
It shows up in the Windows package I have though I'm not configured to test it. It _should_ work as that is a requirement for new features coming into OE SDK (The tests pass on Windows)

Reply:
Yes, need to fix the switchless sample for Windows. I opened an issue https://github.com/openenclave/openenclave/issues/2236

Reply:
`tests/switchless` is cross platform because it has a `get time` function works on both Linux and Windows . We just need to add the same utility function to `samples/switchless`. Ideally, we should expose those cross-platform functions to people who write tests and samples, so we don't have to copy them in each test/sample.

Reply:
@jxyang thanks. In the mean time I will fix the sample in the same way as @anakrish has fixed the tests.